applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -147,6 +168,8 @@ class TrilinosBlockBuilderAndSolver
      */
     TrilinosBlockBuilderAndSolver& operator=(const TrilinosBlockBuilderAndSolver& rOther) = delete;
 
+    // TODO: In order to create a Create method something must be done about the EpetraCommunicator. Maybe ge it from ProcessInfo
```

Comment:
FYI the name of the DataComm should be specified in the settings, then it can be retrieved from the ParallelEnv
This way you can construct an EpetraComm

See e.g. how the distributed_model_part_importer does it when one wants to use a custom communicator

Reply:
Good to know

Reply:
can you please update the comment accordingly?

Reply:
Can you suggest a comment update?

Reply:
@philbucher regarding the DataCommunicator, I like the idea of passing it through the parameters but it would be a slight behavior change: now the EpetraComm's lifecycle is managed by the caller, the Builder&Solver only holds a reference to it (in FluidDynamics it used to be kept by the Python solver, but I'm not sure it is still/always the case).

With the string approach, it should be constructed and managed by the builder and solver itself. I do not think it is a big deal, but it is a change and I don't know if it could be a problem for some users. Any thoughts?